src/README.md

```diff
@@ -266,10 +266,15 @@ file inside your source code repository.
 
 * **UPGRADE_PIP_TO_LATEST**
 
+    {% if spec.version == "2.7" %}
+    Python 2 images are using the latest compatible pip release (<21.0) so this
+    option controls update of setuptools and wheel packages only.
+    {% else %}
     Set this variable to a non-empty value to have the 'pip' program and related
```

Comment:
The Python 2 description is too terse, it's missing "Set this variable to a non-empty value to have the ..." for example.

Reply:
Fixed.